### ISSUE TITLE
Firebase Cloud Messaging (FCM) is deprecated

### ISSUE THREAD

**Issue by luckylittle** [murgo/IrssiNotifier#207]
Sending messages (including upstream messages) with the FCM XMPP and HTTP legacy APIs was deprecated on June 20, 2023, and will be removed in June 2024. IrssiNotifier is using FCM to notify a client app.

**Comment by murgo** [comment]
Indeed. I'm hoping to upgrade the notifications (among other points to upgrade) during April.
